core/shared/src/main/scala/scalaz/zio/Exit.scala

```diff
@@ -1,148 +1,146 @@
 // Copyright (C) 2017-2018 John A. De Goes. All rights reserved.
 package scalaz.zio
 
-import scala.util.{ Failure, Success, Try }
-
 /**
  * An `ExitResult[E, A]` describes the result of executing an `IO` value. The
```

Comment:
Don't forget to update scaladoc in places like this, which quote the code name.

Reply:
:heavy_check_mark: